controller/controller.go

```diff
@@ -30,6 +30,12 @@ import (
 	"github.com/clearlinux/clr-installer/utils"
 )
 
+var (
+	// NetworkPassing is used to track if the latest network configuration
+	// is passing; changes in proxy, etc.
+	NetworkPassing bool
```

Comment:
we could use the model to store this flag, we don't even need to export it anyway.

Reply:
@dorileo But is "the network working" really part of the model of the install?  To me, it was a status of something during execution. But if you feel strongly I can relocate it.